src/main/kotlin/org/springframework/data/r2dbc/core/CriteriaStepExtensions.kt

```diff
@@ -43,3 +43,30 @@ fun Criteria.CriteriaStep.isIn(vararg value: Any): Criteria =
  */
 fun Criteria.CriteriaStep.isIn(values: Collection<Any>): Criteria =
 		`in`(values)
+
```

Comment:
We need to rather introduce an extension to Spring Data Relational instead of growing this one.

Reply:
I filed https://jira.spring.io/browse/DATAJDBC-522 to address the extension part in Spring Data Relational.